lib/wrapped_print.rb

```diff
@@ -51,8 +52,12 @@ def wp(label = nil, pattern: PATTERN, count: COUNT, prefix: nil, suffix: nil, co
 
     private
     def detect_logger_method
-      if WrappedPrint.log_to == :logs
+      case WrappedPrint.log_to
+      when :logs
         Rails.logger.method(WrappedPrint.level)
+      when :logfile
+        @logfile ||= Logger.new("#{Rails.root}/log/wrapped_print.log")
```

Comment:
can you make it configurable? I mean filename.

also a 2 simple specs just to cover this method that it just works!